MRI_geom/SPD_fMRI.py

```diff
@@ -0,0 +1,158 @@
+import networkx as nx
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import time
+from sklearn.svm import SVC
+from sklearn.model_selection import KFold
+from sklearn.metrics import precision_score, accuracy_score
+from sklearn.metrics import f1_score, recall_score, confusion_matrix
+import geomstats.spd_matrices_space as spd_space
+SPACE = spd_space.SPDMatricesSpace(dimension=28)
+
+
+graphs = pd.DataFrame.from_csv("data/train_FNC.csv")
+map_FCN = pd.DataFrame.from_csv("AdditionalInformation/comp_ind_fMRI.csv",
+                                index_col=None)
+map_FCN = map_FCN['fMRI_comp_ind'].to_dict()
+map_FCN_r = {v: k for k, v in map_FCN.iteritems()}
+mapping = pd.DataFrame.from_csv("AdditionalInformation/rs_fMRI_FNC_mapping.csv")
+graph_labels = pd.DataFrame.from_csv(
+                                     "data/train_labels.csv")
```

Comment:
Indent is a bit weird here

Reply:
Same: add a CONSTANT variable a the top of the file for this csv:
`TRAIN_LABELS_CSV = 'data/train_labels.csv'`